demos/browser/app/meeting/meeting.ts

```diff
@@ -474,6 +475,32 @@ export class DemoMeetingApp implements AudioVideoObserver, DeviceChangeObserver
     });
   }
 
+  getNearestMediaRegion(): void {
+    const supportedMediaRegions: Array<string> = ['ap-northeast-1', 'ap-southeast-1', 'ap-southeast-2', 'ca-central-1', 'eu-central-1', 'eu-north-1', 'eu-west-1', 'eu-west-2', 'eu-west-3', 'sa-east-1', 'us-east-1', 'us-east-2', 'us-west-1', 'us-west-2'];
+    const defaultNearestMediaRegion = "us-east-1";
+    new AsyncScheduler().start(
+      async (): Promise<void> => {
+      try {
+        const nearestMediaRegionResponse = await fetch(
+          `https://nearest-media-region.l.chime.aws`,
+          {
+            method: 'GET',
+          }
+        );
+        const nearestMediaRegionJSON = await nearestMediaRegionResponse.json();
+        const nearestMediaRegion = nearestMediaRegionJSON.region;
+        if(nearestMediaRegion !== null || nearestMediaRegion !== '' || supportedMediaRegions.indexOf(nearestMediaRegion) != -1) {
+          (document.getElementById('inputRegion') as HTMLInputElement).value = nearestMediaRegion;
+        }
```

Comment:
I understand that the demo is not auto-formatted with ESLint and Prettier. Please follow the convention in this file.
`} else {`